crane/container.go

```diff
@@ -12,6 +11,8 @@ import (
 	"strconv"
 	"strings"
 	"sync"
+
+	"github.com/flynn/go-shlex"
```

Comment:
do you have a special version of `go fmt`? I don't have the same import ordering